### ISSUE TITLE
Bad concept creation

### ISSUE THREAD

**Issue by hfaghihi15** [HLR/DomiKnowS#288]
Hi, 

As we previously had the notion of `concept("str")` to generate a named subconcept but @auszok has changed it to initializing a variable, we now get different results from `concept1 = concept("concept1")` and `concept1 = concept(name='concept1')` . I think we should fix and change this notion to avoid future problems.

@guoquan @auszok

**Comment by auszok** [comment]
All the graphs where using "name" keyward like:
       char = Concept(name='char')
       word = Concept(name='word')
       phrase = Concept(name='phrase')
       sentence = Concept(name='sentence')

Thus I am using concept("x") in the new lc  definition.

**Comment by hfaghihi15** [comment]
> All the graphs where using "name" keyward like:
> char = Concept(name='char')
> word = Concept(name='word')
> phrase = Concept(name='phrase')
> sentence = Concept(name='sentence')
> 
> Thus I am using concept("x") in the new lc definition.

Yes but actually the name was optional and we mentioned this somewhere in the document. This may cause confusion also when people are trying to work with this. We can either come up with a name for the variable definition or mention this clearly in the documentation and remove the old notes.

**Comment by auszok** [comment]
I checked the examples before I adopted the usage concept(x)  for new lc. I believe most if not all used name when defining concept.
Actually all the example run.

Thus I will suggest that we remove the optional way of defining concept from documentation and required the usage of name, as we anyway did in our examples.

**Comment by hfaghihi15** [comment]
@AdmiralDarius Can you check your documentation for the graph concepts to make sure that it doesn't mention that the `name` property is optional? I remember that we mentioned it before so it may still be somewhere in your new doc.
